Doc/library/http.client.rst

```diff
@@ -461,7 +461,7 @@ also send your request step by step, by using the four functions below.
       This is to avoid premature termination of the read of the request by
       the target server due to malformed encoding.
 
-   .. versionadded:: 3.6
+   .. versionchanged:: 3.6
       Chunked encoding support.  The *encode_chunked* parameter was
       added.
```

Comment:
```suggestion
      Added chunked encoding support and the *encode_chunked* parameter.
```

Clarify that chunked encoding support was _added_ rather than just changed, use consistent active voice, simplify phrasing and fit on one line.